firebase-web/src/main/java/io/spine/web/firebase/FirebaseRecord.java

```diff
@@ -71,19 +76,95 @@ FirebaseDatabasePath path() {
      * @see FirebaseQueryBridge FirebaseQueryBridge for the detailed storage protocol
      */
     void storeTo(FirebaseDatabase database) {
-        final DatabaseReference reference = path().reference(database);
+        DatabaseReference reference = path().reference(database);
         flushTo(reference);
     }
 
+    /**
+     * Writes this record to the given {@link FirebaseDatabase} in a single transaction
+     * (i.e. in a single batch).
+     *
+     * @see FirebaseQueryBridge FirebaseQueryBridge for the detailed storage protocol
```

Comment:
That's weird. How `FirebaseQueryBridge` helps to understand what this method is intended to do? With this thinking approach we could put references `FirebaseQueryBridge` to each method...

Let's discuss in person.